wilsonChan/assignments/treesBFS/lc107/lc107.py

```diff
@@ -1,6 +1,55 @@
 # Problem Statement #
 
-# Given a binary tree, populate an array to represent its level-by-level traversal in reverse order, i.e., the lowest level comes first. You should populate the values of all nodes in each level from left to right in separate sub-arrays.
+# Given a binary tree, populate an array to represent its level-by-level traversal 
+# in reverse order, i.e., the lowest level comes first. You should populate the values of 
+# all nodes in each level from left to right in separate sub-arrays.
+
+'''
+
```

Comment:
Nice overview and visual. Could be helpful to write additional test cases, especially for any edge cases.